src/Be.Vlaanderen.Basisregisters.GrAr.Legacy/SpatialTools/GeoJSONPoint.cs

```diff
@@ -21,6 +21,13 @@ public class GeoJSONPoint
         [JsonProperty(Required = Required.DisallowNull)]
         public string Type { get; set; }
 
+        /// <summary>
```

Comment:
This will change the current contract of GeoJSONPoint and GML will be outputted along the `coordinates`.
I think its better to create a GmlJsonPoint or something.

Reply:
It's not a breaking change, since you add a property. But this will pollute the response with an extra format.